sidebars.js

```diff
@@ -648,11 +648,19 @@ module.exports = {
             "self-managed/platform-deployment/helm-kubernetes/deploy",
             "self-managed/platform-deployment/helm-kubernetes/upgrade",
             {
-              Platforms: [
+              type: "category",
+              label: "Platforms",
+              link: {
+                type: "doc",
+                id: "self-managed/platform-deployment/helm-kubernetes/platforms/platforms",
```

Comment:
Regarding `platforms/platforms`, I'm using the same convention mentioned in the official docs for [Category index](https://docusaurus.io/docs/2.0.1/sidebar/autogenerated#category-index-convention).